fetch/api/response/multi-globals/current/current.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
```

Comment:
Would it be possible to move this file in "fetch/api/resources" folder?
That way, it could be merged with the request current.html